examples/tutorials/Scanpy.ipynb

```diff
@@ -21,37 +21,44 @@
     "\n",
```

Comment:
<p>The markdown is broken here (should be a header)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3834/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>